src/Surfnet/StepupMiddleware/CommandHandlingBundle/Resources/config/processors.yml

```diff
@@ -52,6 +52,11 @@ services:
             - "" # Fallback locale
             - "" # Self service url is set in bundle extension
 
+    surfnet_stepup_middleware_command_handling.service.second_factor_displayname_resolver:
+        public: false
+        class: Surfnet\StepupMiddleware\CommandHandlingBundle\Identity\Service\SecondFactorDisplayNameResolverService
+        arguments: ['%enabled_generic_second_factors%']
+
```

Comment:
I dont think this service belongs here perse. Granted it's used by a processor, but the service itself is not a processor.